care/facility/api/serializers/patient.py

```diff
@@ -421,8 +423,18 @@ def save(self, **kwargs):
 
 
 class PatientNotesSerializer(serializers.ModelSerializer):
+    NOTE_EDIT_WINDOW = 30 * 60  # 30 minutes
+
     facility = FacilityBasicInfoSerializer(read_only=True)
     created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
+    edit_window = serializers.SerializerMethodField(read_only=True)
+
+    def get_edit_window(self, obj):
+        return self.NOTE_EDIT_WINDOW
```

Comment:
If this value is only used in the viewset then why is it stored in the serializer?  Maintain a Variable for this in FE as well, Returning a constant value for every item seems kinda extra. Storing a last editable datetime in the notes object is a better approach than this.

Reply:
Moved it to the model and used `editable_until` logic.